Pipfile

```diff
@@ -9,4 +9,4 @@ renku = {editable = true, extras = ["all"], path = "."}
 [dev-packages]
 
 [requires]
-python_version = "3.7"
+python_version = "3.8"
```

Comment:
is this intended? I don't think we want to force 3.8?

Reply:
It's intended yes. I see no reasons why not, do you? Want me to roll it back?

Reply:
In my view this should be the default supported version - I think we consider 3.8 experimental, no?

Reply:
I also believe it makes sense to stay for a bit longer on `3.7` while just providing `3.8` compatibility.